src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1600,14 +1648,45 @@ void NetworkOPsImp::pubServer ()
     if (!mSubServer.empty ())
     {
         Json::Value jvObj (Json::objectValue);
-        auto const& feeTrack = app_.getFeeTrack();
 
-        jvObj [jss::type]          = "serverStatus";
+        ServerFeeSummary f{app_.openLedger().current()->fees().base,
+            app_.getTxQ().getMetrics(*app_.openLedger().current()),
+            app_.getFeeTrack()};
+
+        // Need to cap to uint64 to uint32 due to JSON limitations
+        auto clamp = [](std::uint64_t v)
```

Comment:
can you use the impl in beast for this?   https://github.com/vinniefalco/Beast/blob/master/include/beast/core/detail/clamp.hpp